components/webfield/ProgramChairConsole.js

```diff
@@ -1101,7 +1133,7 @@ const ProgramChairConsole = ({ appContext, extraTabs = [] }) => {
               <Tab
                 id={fieldAttrs.field}
                 key={fieldAttrs.field}
-                active={activeTabId === fieldAttrs.field ? true : undefined}
+                active={activeTabId === `#${fieldAttrs.field}` ? true : undefined}
```

Comment:
> causing the submissionContentFields panel to fail to load when url already has the hash

@xkopenreview I think this was because we weren't adding `#` in front of the field here. I reproduced it in my local and the panel loads now.